README.md

```diff
@@ -38,11 +38,11 @@ If you are using Maven with [BOM][libraries-bom], add this to your pom.xml file
 If you are using Maven without BOM, add this to your dependencies:
 
 ```xml
-    <dependency>
-      <groupId>com.google.cloud</groupId>
-      <artifactId>google-cloud-logging</artifactId>
-      <version>1.101.0</version>
-    </dependency>
+<dependency>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-logging</artifactId>
+  <version>1.101.1-SNAPSHOT</version>
```

Comment:
This should be 2.0.0-SNAPSHOT.

Reply:
This will be updated by the release tooling and actually shouldn't be a SNAPSHOT in the README